catroid/build.gradle

```diff
@@ -106,6 +106,7 @@ android {
     }
 
     defaultConfig {
+        testApplicationId 'org.catrobat.catroid.test'
```

Comment:
why do we that?

Reply:
This is because the flavored tests wouldn't compile without that magic line.
Otherwise the .R files would be generated with package name org.catrobat.catroid.createatschool.test.R
However, the "main tests" are using org.catrobat.catroid.test.R. This leads to compile errors, as the .R file wouldn't exist.
This is the same as in the phiro-branch (had to do that again here, as the branch isn't merged yet)

Reply:
ok, good to know.